modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/list/List.tsx

```diff
@@ -256,6 +296,48 @@ function ListSection({
 		});
 	};
 
+	let updateButton = (
+		<ClayButtonWithIcon
+			aria-label={Liferay.Language.get('assign-field')}
+			displayType="secondary"
+			onClick={openSelectFieldModal}
+			symbol="plus"
+			title={Liferay.Language.get('assign-field')}
+		/>
+	);
+
+	if (field) {
```

Comment:
This seems functional, but it's not very React-y. A cleaner solution would be a new "private" component, something like `<UpdateButton>`, that would internally `return field ? <first version> : <second version>`

Reply:
Done!